tests/Unit/Domain/Test_FaceNormal.cpp

```diff
@@ -67,6 +80,120 @@ void check(const Map& map,
   }
 }
 
+template <size_t Dim>
+std::unordered_set<Direction<Dim>> get_directions();
+
+template <>
+std::unordered_set<Direction<1>> get_directions<1>() {
+  return std::unordered_set<Direction<1>>{Direction<1>::upper_xi()};
+}
+
+template <>
+std::unordered_set<Direction<2>> get_directions<2>() {
+  return std::unordered_set<Direction<2>>{Direction<2>::upper_xi(),
+                                          Direction<2>::lower_eta()};
+}
+
+template <>
+std::unordered_set<Direction<3>> get_directions<3>() {
+  return std::unordered_set<Direction<3>>{Direction<3>::upper_xi(),
+                                          Direction<3>::lower_eta(),
+                                          Direction<3>::lower_zeta()};
+}
+
+template <size_t Dim>
+void check_time_dependent(
+    const ElementMap<Dim, Frame::Grid>& logical_to_grid_map,
+    const std::unique_ptr<CoordinateMapBase<Frame::Grid, Frame::Inertial, Dim>>&
+        grid_to_inertial_map,
+    const std::unique_ptr<CoordinateMapBase<Frame::Logical, Frame::Inertial,
+                                            Dim>>& logical_to_inertial_map,
+    const double time,
+    const std::unordered_map<
+        std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+        functions_of_time) {
+  const Mesh<Dim - 1> interface_mesh{3, Spectral::Basis::Legendre,
```

Comment:
If I'm reading this right, then in 1D, you are making a Mesh<0> with 3 grid points... which I think is 2 more than should be there